test/simulation/install.sh

```diff
@@ -8,28 +8,34 @@ delete_dir() {
 	return 0
 }
 
-
 temp_gopath=$PWD/temp/go
 temp_lndpath=${temp_gopath}/src/github.com/lightningnetwork/lnd
-
+LND_COMMIT="v0.6.1-beta-rc2"
 if [ -f ${temp_lndpath}/lnd-debug ]
 then
+  LND_VERSION=$(${temp_lndpath}/lnd-debug --version)
+else
+  LND_VERSION=""
+fi
+
+if [[ $LND_VERSION == *"$LND_COMMIT" ]]; then
     echo "lnd already installed"
 else
+    echo "deleting temporary gopath directory"
     delete_dir ${temp_gopath}
 
-    GO111MODULE=off GOPATH=${temp_gopath} go get -u github.com/golang/dep/cmd/dep
+    echo "getting btcd..."
     GO111MODULE=off go get -u github.com/btcsuite/btcd
 
     echo "starting lnd clone..."
-    if ! git clone -b resolver-cmd+simnet-ltcd https://github.com/ExchangeUnion/lnd.git ${temp_gopath}/src/github.com/lightningnetwork/lnd > /dev/null 2>&1; then
+    if ! git clone -b v0.6.1-beta-rc2 --depth 1 https://github.com/lightningnetwork/lnd ${temp_lndpath} > /dev/null 2>&1; then
```

Comment:
should be able to use `$LND_COMMIT` here

Reply:
Yeah that makes sense. It won't work if we have to go back to using a commit hash instead of a tag, but in that case this line needs to change anyway. I'll go ahead and make this change and rename the variable to `$LND_TAG`.